enum.c

```diff
@@ -4179,13 +4179,137 @@ enum_uniq(VALUE obj)
+ *
+ * === Methods for Searching and Filtering
+ *
+ * These methods return elements that meet a specified criterion.
+ *
+ * #find, #detect::              Returns an element selected by the block.
+ * #find_all, #filter, #select:: Returns elements selected by the block.
+ * #grep::                       Returns elements selected by a given object or block.
+ * #grep_v::                     Returns elements selected by a given object or block.
+ * #find_index::                 Returns the index of an element selected by a given object or block.
+ * #reject::                     Returns elements not rejected by the block.
+ * #uniq::                       Returns elements that are not duplicates.
+ *
+ * === Methods for Sorting
+ *
+ * These methods return elements in sorted order.
+ *
+ * #sort::    Returns the elements, sorted by <tt><=></tt> or the given block.
+ * #sort_by:: Returns the elements, sorted by the given block.
+ *
+ * === Methods for Iterating
+ *
+ * #each_entry::       Calls the block with each successive element
+ *                     (slightly different from #each).
+ * #each_with_index::  Calls the block with each successive element and its index.
+ * #each_with_object:: Calls the block with each successive element and a given object.
+ * #each_slice::       Calls the block with successive non-overlapping slices.
+ * #each_cons::        Calls the block with successive overlapping slices.
+ *                     (different from #each_slice).
+ * #reverse_each::     Calls the block with each successive element, in reverse order.
+ *
+ * === Other Methods
+ *
+ * #map, #collect::             Returns objects returned by the block.
+ * #filter_map::                Returns truthy objects returned by the block.
+ * #flat_map, #collect_concat:: Returns flattened objects returned by the block.
+ * #reduce, #inject::           Returns the object formed by combining all elements.
+ * #sum::                       Returns the sum of the elements, using method +++.
+ * #zip::                       Combines each element with elements from other enumerables;
+ *                              returns the n-tuples or calls the block with each.
+ * #cycle::                     Calls the block with each element, cycling repeatedly.
+ *
+ * == Usage
+ *
+ * To use module \Enumerable in a collection class:
+ * - Include it:
+ *     include Enumerable
+ * - Implement method <tt>#each</tt>
+ *   which must yield successive elements of the collection.
+ *   This method will be called by almost any \Enumerable method.
+ * - Implement method <tt><=></tt> (or just inherit
+ *   {Object#<=>}[https://ruby-doc.org/core/Object.html#method-i-3C-3D-3E]).
+ *   The method will be called by many \Enumerable methods, including
+ *   <tt>#max</tt>, <tt>#min</tt>, and <tt>#sort</tt>.
+ *
+ * == \Enumerable in Ruby Core Classes
+ * Some Ruby classes include module \Enumerable:
```

Comment:
"include *the* module `Enumerable`", or "include `Enumerable`"

Reply:
Fixed. "include Enumerable"